api/spec/integration/api/v2/platform/tax_rates_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'swagger_helper'
+
+describe 'Tax Rates API', swagger: true do
+  include_context 'Platform API v2'
+
+  resource_name = 'Tax Rate'
+  options = {
+    include_example: 'zone,tax_category',
+    filter_examples: [{ name: 'filter[zone_id_eq]', example: '3' },
+                      { name: 'filter[amount_gt]', example: '0.05' },
+                      { name: 'filter[tax_category_id_eq]', example: '1' }]
```

Comment:
@aleksandarpetrushev to make this filtering work you need to add whitelisted ransack attributes - see my PR for Line Items https://github.com/spree/spree/pull/11429/files